Translation/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  translation-tgi-service:
+    image: ghcr.io/huggingface/text-generation-inference:2.3.1-rocm
+    container_name: translation-tgi-service
+    ports:
+      - "${TRANSLATION_TGI_SERVICE_PORT:-8008}:80"
+    volumes:
+      - "/var/lib/GenAI/translation/data:/data"
```

Comment:
The volume here should be './data' by default, since we don't offer default data folder.